xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspCsvRecord.java

```diff
@@ -61,7 +61,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "AspspAdapterConfigRecord{" +
+        return "de.adorsys.xs2a.adapter.registry.AspspAdapterConfigRecord{" +
```

Comment:
old name of class

Reply:
Good catch. Regenerated the `toString` method.